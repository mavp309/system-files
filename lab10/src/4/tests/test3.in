100000000 100000000
999999999 999999999
