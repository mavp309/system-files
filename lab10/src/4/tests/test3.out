1100000000099999999
